public/components/trace_analytics/components/common/plots/service_map.tsx

```diff
@@ -81,6 +83,10 @@ export function ServiceMap({
   filterByCurrService?: boolean;
   includeMetricsCallback?: () => void;
   mode?: string;
+  filters: FilterType[];
+  setFilters:
+    | React.Dispatch<React.SetStateAction<FilterType[]>>
+    | ((filters: FilterType[]) => void);
```

Comment:
curious why are both needed?

Reply:
Thought i needed to cover other conditions, removed it and seems to work fine.
Thanks